source/releasenotes/2024-05-10-sendgrid-transactional-emails.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Transactional emails now sent via SendGrid API"
+published_date: "2024-05-10"
+categories: [new-feature, wordpress, drupal]
+---
+Beginning this week, Pantheon will start transitioning individual WordPress and Drupal sites to use Sendgrid's API for the delivery of outgoing email initiated by these CMSes. We expect all sites to be fully transitioned by May 31, 2024. This change is happening at the platform level, not within the CMS code or configuration, and therefore requires no action by customers. 
+
+<h3>Reason for transition</h3>
```

Comment:
```suggestion
## Reason for transition
```

We should stick with markdown syntax for headings in markdown files. I'm also switching it to an h2 since the title is the h1